README.md

```diff
@@ -1 +1,79 @@
-# Book-Store
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Book-Store
+"Bookstore" is an MVP website that allows you to:
```

Comment:
- [ ] You didn't add a license file in your project. Please add license using these [guidlines](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository) :+1: because license file is very important these days you can understand :+1:
- [ ] After adding the license file please add the link of your license file here in your readme file to make it easily accessible for viewers of your repository :blush:

Reply:
Updated Readme with newly created license file. Thanks for reviewing the project